ui/app/src/components/ContentTypeManagement/utils.ts

```diff
@@ -260,8 +262,15 @@ export function reverseTypeFieldValuesObject(
 					properties[property] = mergedProperties[property];
 					continue;
 				}
+				const fieldDescriptor = datasourceFields[property];
+				if (!fieldDescriptor) {
+					// Drop unknown/obsolete property not present in descriptor
+					continue;
+				}
```

Comment:
What's an example of this?

Reply:
This update was after a rabbit review, to guard against a possible NPE if there's a property of the field that is not in the descriptor.